### ISSUE TITLE
django.db.utils.ProgrammingError

### ISSUE THREAD

**Issue by carolinetm82** [healthsites/healthsites#1341]
Hi,
Me and a friend of mine are trying to do the README 

We installed Docker, we executed the `make run-docker-osm` command before doing the README but we encountered the same problem again after trying to execute `Make deploy` command. We have the following error:

_django.db.utils.ProgrammingError: relation"osm_healthcare_facilities_node" does not exist
LINE3: ...sm_id,'-node') as row, 'node' as osm_type, * from osm_health..._

_make: ***[Makefile:112: migrate] Error 1_

Thank you in advance for helping us to resolve this issue

**Comment by meomancer** [comment]
hi @carolinetm82 
I update readme
https://github.com/healthsites/healthsites#how-to-setup
and how to overcome that issue

**Comment by meomancer** [comment]
please close if it is working

**Comment by carolinetm82** [comment]
Hi Irwan,

Thank you for updating the README file. I also saw you made the healthsites
shapefiles available for download on the website, thank you, that will be
useful for my team.
I will test your code and close if it works.

Best regards,
Caroline CAO

Le jeu. 7 mai 2020 à 02:57, Irwan Fathurrahman <notifications@github.com> a
écrit :

**Comment by carolinetm82** [comment]
Hi Irwan,
Sorry for being late to respond to you. I tested your updated readme, the
first 'make deploy' worked but the second 'make deploy' caused some
technical issues on my virtual machine and in the end I had to interrupt
it. But it's okay because my purpose was to extract the data and you've
already done it by making them available to download on shp or csv on your
website.
Thank you.

Le jeu. 7 mai 2020 à 02:57, Irwan Fathurrahman <notifications@github.com> a
écrit :
